examples/great3/cgc.yaml

```diff
@@ -16,9 +16,11 @@
 #    and/or other materials provided with the distribution.
 
 # The Great3 control-ground-constant branch configuration file
-# Note: This configuration will not produce images _identical_ to the ones used for Great3, 
+# Note: This configuration will not produce images _identical_ to the ones used for Great3,
 # since the order of random number generation is different.  However, it is intended to
 # produce an _equivalent_ simulation as the one produced for Great3.
+modules:
```

Comment:
The DES examples are a more complete set of examples that use the modules field in various ways.  cf. the PR for #654.  I've called both the DES and the Great3 examples "Advanced Simulations" [here](https://github.com/GalSim-developers/GalSim/wiki/Tutorials) and mention the top-level modules field as a significant feature of both.

I'm not averse to including the modules in the main set of numbered demos.  I think the natural place will likely be demo13.  But that will have to wait until the chromatic stuff is folding into the config framework, which I haven't done yet.